IPs/RSKIP419.md

```diff
@@ -0,0 +1,229 @@
+
+
+The process consists of the following 3 steps:
+
+##### 1. Funding transaction creation
+
+As soon as the proposed Powpeg has been voted (and saved in a new Bridge storage entry, key `proposedFederation`), the Bridge will create a SPV funding transaction for it.
+
+This transaction is similar to a peg-out where the recipient is the proposed Powpeg Bitcoin address, and the amount to receive is calculated based on the minimum amount of satoshis required to spend an UTXO by it.
+
+Once the transaction is created, it has to be stored in the set for peg-outs waiting for confirmations, and the sighash has to be stored in the pegout index so the change could be registered.
+
+Also, a new value in the storage should be added to identify this special kind of transaction:
+    
+    KEY: "FUND_TX_HASH_UNSIGNED"
+    VALUE: hash of SVP funding tx without signatures
+
+##### 2. Funding transaction confirmation
+
+The SVP funding transaction goes through the usual peg-out confirmation process. Once the transaction accumulates at least 4_000 Rootstock blocks (for mainnet) it should be ready to be moved from the current set to the waiting for signatures set.
+
+##### 3. Funding transaction signing
+
+The funding transaction should be signed as any other peg-out. Once it is fully signed it should be removed from the set and a release_btc event should be emitted.
+Eventually, when the funding transaction is registered in the Bridge, the Proof transaction could be created.
+
+
+### Proof transaction creation
+
+Once the funding transaction is registered, the Proof will be created for the proposed Powpeg to sign and broadcast it so it can prove it can spend funds.
+
+##### Funding transaction registration
+The funding transaction should be registered through the registerBtcTransaction method.
+For that purpose, the method will be modified to be able to identify it.
+It should verify:
+- SVP period is ongoing
+(i.e. proposedFederation exists & CURRENT_RSK_BLOCK < PROPOSED_FED_CREATION_BLOCK_NUMBER + SVP_PERIOD)
+- Transaction hash without signatures matches the record in the FUND_TX_HASH_UNSIGNED storage entry
+
+Once everything is verified, the transaction will be registered and the transaction hash will be saved in a new storage entry, FUND_TX_HASH_SIGNED.
+The funding transaction hash signed being saved means that the proof Transaction can be created.
+
+**Important remark**: This method will be public, meaning any user could call it.
+
+#### Proof transaction creation or validation process failure
+
+A new method, `processPowpegChangeValidation` will be called from the `updateCollections` method.
+
+The `processPowpegChangeValidation` will perform some actions if a proposedFederation exists depending on this two scenarios:
+- SVP period ended. This implies the validation was not successful.
```

Comment:
```suggestion
- SVP period ended and the proof transaction was not received. This implies the validation was not successful.
```

Reply:
I'm not sure about this one.
If the SVP period ended and the proposedFederation still exists, it implies that the proof tx was not received. Remember that after the proof tx is received, the proposedFederation won't exist any more.

wdyt?

Reply:
`SVP period ended. This implies the proof transaction was not received, and therefore that the validation was not successful.`

wdyt?